tor/core/posts.py

```diff
@@ -103,10 +104,9 @@ def process_post(new_post, config):
         content_type = 'video'
         content_format = config.video_formatting
     else:
-        # how could we get here without fulfilling one of the above
-        # criteria? Just remember: the users will find a way.
-        content_type = 'Unknown'
-        content_format = 'Formatting? I think something went wrong here...'
+        # This means we pulled from a subreddit bypassing the filters.
+        content_type = 'Other'
+        content_format = config.other_formatting
```

Comment:
Is the `config.other_formatting` value being set somewhere? It looks like it requires 1 of 2 things still:

- update to `tor_core` repo for additional value
- update to initialization of this bot in particular in order to set this value

Reply:
Both of which are done over in https://github.com/TranscribersOfReddit/tor_core/pull/8/files

Reply:
Oops! You're right. It was even put in the description of this PR. 😅

Reply:
![nodding](https://user-images.githubusercontent.com/5179553/31503063-a46733e6-af3c-11e7-912c-064cc0e035ef.gif)